skimage/restoration/_denoise.py

```diff
@@ -885,6 +892,8 @@ def denoise_wavelet(image, sigma=None, wavelet='db1', mode='soft',
 
     Examples
     --------
+    >>> import pytest
+    >>> _ = pytest.importorskip('pywt')
```

Comment:
Found this solution here
https://github.com/scikit-image/scikit-image/blob/cb353695fc23eb2a19a58c14d25485e0eeae289b/skimage/feature/_fisher_vector.py#L82-L83 

Would be nice to hide these lines somehow or skip this doctest another way but not sure how.